### PR TITLE
docs kickoff

### PR DESCRIPTION
**Pull request by janlegner**

### CODE REVIEW

File: api/src/handlers/admin_score_upload.rs [hunk 1]
```diff
@@ -13,19 +14,29 @@ use warp::{
...
     epoch: i32,
     components: String,
     component_weights: String,
     ui_id: String,
 }
 
+#[utoipa::path(
+    post,
+    tag = "Admin",
+    operation_id = "List validators",
```
Comment (butonium) [inline api/src/handlers/admin_score_upload.rs:33]: is this correct?

**Review by butonium — CHANGES_REQUESTED** [review]
when calling /docs I receive this error
`Error: Invalid reference token: UptimeRecord
    at o.get (https://public.marinade.finance/redoc.v2.0.0.standalone.js:8:261581)
    at Yc (https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:71461)
    at https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:76462
    at Yc (https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:72882)
    at https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:76862
    at Array.forEach (<anonymous>)
    at https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:76791
    at Yc (https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:72882)
    at Yc (https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:71505)
    at eu (https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:76119)`
Screenshot 2023-04-03 at 12 26 02

**Comment by janlegner** [comment]
> when calling /docs I receive this error `Error: Invalid reference token: UptimeRecord at o.get (https://public.marinade.finance/redoc.v2.0.0.standalone.js:8:261581) at Yc (https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:71461) at https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:76462 at Yc (https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:72882) at https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:76862 at Array.forEach (<anonymous>) at https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:76791 at Yc (https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:72882) at Yc (https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:71505) at eu (https://public.marinade.finance/redoc.v2.0.0.standalone.js:39:76119)` Screenshot 2023-04-03 at 12 26 02

thx, that is fixed

File: api/src/handlers/admin_score_upload.rs (hunk 1 above)
Comment (janlegner) [reply api/src/handlers/admin_score_upload.rs]: fixed

**Review by butonium — APPROVED** [review]

**Pull request merged**